core/admin/start.py

```diff
@@ -37,11 +37,11 @@ def test_DNS():
             try:
                 result = resolver.resolve('example.org', dns.rdatatype.A, dns.rdataclass.IN, lifetime=10)
             except Exception as e:
-                log.critical("Your DNS resolver at %s is not working (%s). Please use another resolver or enable unbound via https://setup.mailu.io.", ns, e);
+                log.critical("Your DNS resolver at %s is not working (%s). Please see https://mailu.io/master/faq.html#the-admin-container-won-t-start-and-its-log-says-critical-your-dns-resolver-isn-t-doing-dnssec-validation", ns, e);
```

Comment:
I think we could (and should) use /version here.
Just change the message
```
log.critical("Your DNS resolver at %s is not working (%s). Please see https://mailu.io/%s/faq.html#the-admin-container-won-t-start-and-its-log-says-critical-your-dns-resolver-isn-t-doing-dnssec-validation", ns, e, version);
```

And load the file on line ~8:
```
try:
    version = '.'.join(open('/version').read().split('.')[:2])
except:
    version = 'master'
```

Reply:
As long the user is not on an old version of Mailu (for which there is not a documentation site), this should work.

Reply:
IMHO that's all stuff that socrates (or its replacement) should do/deal with. Odds are this isn't the only place that would benefit from knowing the version number